Gateway/Http/Client/TransactionCapture.php

```diff
@@ -124,4 +128,19 @@ private function placeMultipleCaptureRequests(Modification $service, $requestCon
 
         return $response;
     }
+
+    /**
+     * Copy data from the request to the response. This data will be used later when handling the response
+     *
+     * @param array $response
+     * @param array $request
+     * @return array
+     */
+    private function copyParamsToResponse(array $response, array $request): array
+    {
+        $response[self::CAPTURE_AMOUNT] = $request['modificationAmount']['value'];
+        $response[self::ORIGINAL_REFERENCE] = $request['originalReference'];
```

Comment:
Should we check if these variable are there before assigning them?

Reply:
I can add a check just in case, but since they will always be set in the `CaptureDataBuilder` and even if they are not set, the adyen capture request will fail, which will cause this code snippet to be skipped, I'm not sure if such a check would be worth it.